spec/io/io_spec.rb

```diff
@@ -32,6 +32,15 @@
         df = Daru::DataFrame.from_csv 'spec/fixtures/sales-funnel.csv'
         expect(df.vectors.to_a).to eq(%W[Account Name Rep Manager Product Quantity Price Status])
       end
+
+      it "works for urls (header comparison for mock data)" do
+        WebMock.stub_request(:get,'https://raw.githubusercontent.com/anshuman23/ruBayes/master/test/sample2.csv').
+          to_return(status: 200, body: File.read('spec/fixtures/scientific_notation.csv'))
```

Comment:
Please, context + stubbing in before of that context. BTW, with stubs there's no need to use real (and long URL), some `http://dumm/some.csv` will be enough.